ttw/ttw-advanced-2.rst

```diff
@@ -1 +1 @@
-.. include:: ../theming/ttw-advanced-2.rst
```

Comment:
I am not sure this is a good idea :)

Using ``:include`` is tricky, the tone, voice and audience may differ also the style of writing.
This can lead to weird and confusing documentation.

Even more important this will break in the future if people try to build only the ttw docs.
It may work now, but not in the future.

Reply:
@svx I tend to agree with you. I got this change via a merge. What's the solution, though? Duplicate the chapter and recommend diffing to keep it in sync? We have the same issue with two `dexterity.rst` files which seems to be pretty much identical except for formatting.

Reply:
I usually duplicate and change the file then to my needs, like I make sure to use the same style-guide, voice of tone and same rst formating and all such stuff.
Including a file with a different rst formating, like headers for example can produce some 'fancy weird docs'  :)

Also I check if the copied file is OK in terms of content, do I have everything ? Does the user knows all the terms, I had it some time in the training docs, that ppl used ``include`` and suddenly they had terms which were never used before in this training in the docs, because of the include.

One other point is we do not know if content will change/move or will be renamed. 
This can lead to broken docs or missing content :)